### ISSUE TITLE
allow users to customize author order on books

### ISSUE THREAD

**Issue by stopregionblocking** [bookwyrm-social/bookwyrm#2137]
there doesn't seem to be a way to control or predict the order authors are listed in for books with multiple authors. this can be a problem when, for example, the author of a foreword or introduction is listed before the author of a main body of a work, because in the home feed only the first author is displayed. at first i thought bookwyrm would lisy the newest added author in an edit as the new first author, but this doesn't seem to be correct. all this is on the main/flagship instance.

**Comment by mouse-reeve** [comment]
Yeah! This would be a good thing to do along with the issue out there somewhere or other for adding types of authors
